data-plane/dispatcher/pom.xml

```diff
@@ -156,6 +156,12 @@
                     <include>**</include>
                   </includes>
                 </filter>
+                <filter>
```

Comment:
@pierDipi Why is it a `filter` rather than a project `dependency`?

Reply:
it is a dependency but any class that isn't actually referenced in the final uber Jar will be removed, so when Kafka consumer dynamically asks for a class (uses reflection) it won't find it.